### ISSUE TITLE
Misspelled properties are not reported

### ISSUE THREAD

**Issue by exogen** [exogen/jest-styled-components-stylelint#1]
This is because we're passing stylelint a fragment like:

```css
color: blue;
```

Which doesn't include the selector, like:

```css
.foo {
  color: blue;
}
```

We'll need to wrap the output styles with the component's selector for stylelint to fully work. This could potentially require adjusting the line numbers and indentation so they still match up with the user's input.
